crates/starknet-os/src/execution/deprecated_syscall_handler.rs

```diff
@@ -15,6 +15,7 @@ use crate::cairo_types::syscalls::{
     TxInfo,
 };
 use crate::starknet::starknet_storage::PerContractStorage;
+use crate::execution::constants::{VALIDATE_BLOCK_NUMBER_ROUNDING, VALIDATE_TIMESTAMP_ROUNDING};
```

Comment:
We already discussed this, but these should be able to come from the constants in `cairo-lang`. We should do that if at all possible, but if not we should leave a comment about that here.